region/lowernorfair/east/The Worst Room In The Game.json

```diff
@@ -861,28 +903,53 @@
       "link": [3, 5],
       "name": "Base",
       "requires": [
-        "h_canNavigateHeatRooms",
         {"or": [
           {"and": [
-            {"or": [
-              "canTrickyJump",
-              {"enemyKill": {
-                "enemies": [["Yellow Space Pirate (standing)"]],
-                "explicitWeapons": ["ScrewAttack"]
-              }}
-            ]},
+            {"enemyKill": {
+              "enemies": [["Yellow Space Pirate (standing)"]],
+              "explicitWeapons": ["ScrewAttack"]
```

Comment:
Supers and cwip can both also kill the pirate pretty fast. anything less is just the same speed as jumping over it looks like to me.

Reply:
Single charge shot kill with a double hit isn't something I want to include at base difficulty.  You can trickyJump over the pirate instead with another strat.